pyramid_oereb/lib/renderer/extract/templates/xml/extract.xml

```diff
@@ -73,11 +73,9 @@
     %if extract.qr_code:
         <data:QRCode>${extract.qr_code}</data:QRCode>
     %endif
-    %if extract.general_information:
-        <data:GeneralInformation>
-            <%include file="multilingual_text.xml" args="text=extract.general_information"/>
-        </data:GeneralInformation>
-    %endif
+    %for general_information in extract.general_information:
```

Comment:
why did you remove the condition?
As far as I remember, general information is never empty. But I might be wrong

Reply:
The difference is that it is now a list (because it comes from the DB and we take the whole table) even though there should be exactly one element there. So I changed the `if` condition for a `for` loop. I think it's the case for many other things in the extract (lists with only one element, always).
But maybe we should implement it so that is can only have one element (-> would not be a list anymore)? I am not sure at which "layer" (source, reader, etc) it would best to do it but we can do it for sure. What do you think?

Reply:
I think it is at least one element but it could be a list. So that is fine. But as you said, we might want to check elsewhere that there is at least one element or throw an error.
But that is not a high priority.

@vvmruder do we want to put this on a 'beautify and check exceptions' issue list once the main work is done?

Reply:
@voisardf I think we should fix this in place now. And your suggestion is good. Lets implement a throw to assure minimal one element is there.